src/links/dto/create-link.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { IsUrl, IsUUID } from 'class-validator';
+
+export class CreateLinkDto {
+  @IsUrl()
+  shortUrl: string;
+
+  @IsUrl()
+  originalUrl: string;
+
+  @IsUUID()
```

Comment:
**praise**: `@IsUUID()` is a great choice for validating `userId`, ensuring it follows the UUID format.